installer/delta-engine/delta-engine.yaml

```diff
@@ -0,0 +1,189 @@
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: LambdaSQSPolicy
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - ec2:*
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - s3:*
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - sqs:*
+                Resource:
+                  - !GetAtt SQSAssetShippingDone.Arn
+                  - !GetAtt SQSAssetMappingDone.Arn
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource: !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/svc-${Environment}-asset-shipper:*"
+
+  svcassetshipper:
+    Type: AWS::Serverless::Function
+    Properties:
+      FunctionName: !Sub "svc-${Environment}-asset-shipper"
+      CodeUri:
+        Bucket: !Ref S3BUCKETNAME
+        Key: !Sub ${RELEASEVERSION}/${Environment}/deltaengine/asset-sender-1.0-SNAPSHOT.jar
+      Description: Lambda function for asset shipping and delta engine.
+      MemorySize: 512
+      Timeout: 900
+      Handler: com.paladincloud.assetsender.AssetDoneHandler
+      Runtime: java21
+      Architectures:
+        - x86_64
+      EphemeralStorage:
+        Size: 512
+      Environment:
+        Variables:
+          ASSET_SHIPPER_DONE_SQS_URL: !Ref SQSAssetShippingDone
+          JAVA_TOOL_OPTIONS: '--enable-preview'
+          SUBSYSTEM_NAME: ASSET_SHIPPER
+          ENABLED_DELTA_ENGINE_PLUGINS: gcp
+          ASSET_MAPPING_DONE_QUEUE_URL: !Ref SQSAssetMappingDone
+      EventInvokeConfig:
+        MaximumEventAgeInSeconds: 21600
+        MaximumRetryAttempts: 2
+      PackageType: Zip
+      Role: !GetAtt DeltaEngineLambdaExecutionRole.Arn # Attach the role to the Lambda function
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref SecurityGroup
+        SubnetIds:
+          - !Ref Subnet1
+          - !Ref Subnet2
+        Ipv6AllowedForDualStack: false
+      Events:
+        SQS1:
+          Type: SQS
+          Properties:
+            Queue: !GetAtt SQSAssetMappingDone.Arn
+            BatchSize: 10
+      RuntimeManagementConfig:
+        UpdateRuntimeOn: Auto
+
+  LambdaLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties: 
+      LogGroupName: !Sub "/aws/lambda/svc-${Environment}-asset-shipper"
+      RetentionInDays: 14
+
+  SQSAssetShippingDone:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "svc-${Environment}-asset-shipping-done.fifo"  # Use !Sub for environment substitution
```

Comment:
At another revision, we should feed the queue name from the service configuration in config.yaml. It's standard queue name should be ${Environment}-{value_from_config}.

for this revision, you can correct it to ${Environment}-svc-asset-delta-engine-done.fifo